scrapers/HentaiDude.yml

```diff
@@ -0,0 +1,42 @@
+name: "HentaiDude"
+
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - hentaidude.com/
+    scraper: sceneScraper
+
+sceneByName:
+  action: scrapeXPath
+  queryURL: https://hentaidude.com/?s={}
+  scraper: sceneSearch
+
+sceneByQueryFragment:
+  action: scrapeXPath
+  queryURL: "{url}"
+  scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="title-container"]/h2
+      URL: //meta[@property="og:url"]/@content
+      Image: //meta[@property="og:image"]/@content
+      Movies:
+        Name:
+          selector: //meta[@property="og:title"]/@content

```

Comment:
This movie scraper might need an additional regex, right now it doesn't clean `- Hentaidude.com` from the end